packages/web/src/components/gcds-notice/gcds-notice.tsx

```diff
@@ -0,0 +1,125 @@
+import { Component, Element, Host, Prop, State, h } from '@stencil/core';
+import { assignLanguage, observerConfig, logError } from '../../utils/utils';
+import i18n from './i18n/i18n';
+
+@Component({
+  tag: 'gcds-notice',
+  styleUrl: 'gcds-notice.css',
+  shadow: true,
+})
+export class GcdsNotice {
+  @Element() el: HTMLElement;
+
+  /**
+   * Props
+   */
+
+  /**
+   * Set notice type.
+   */
+  @Prop() type!: 'danger' | 'info' | 'success' | 'warning';
+  validateType() {
+    if (!this.type) {
+      this.errors.push('type');
+    } else if (this.errors.includes('type')) {
+      this.errors.splice(this.errors.indexOf('type'), 1);
+    }
+  }
+
+  /**
+   * Set the notice title.
+   */
+  @Prop() noticeTitle!: string;
+  validateNoticeTitle() {
+    if (!this.noticeTitle || this.noticeTitle === '') {
+      this.errors.push('noticeTitle');
+    } else if (this.errors.includes('noticeTitle')) {
+      this.errors.splice(this.errors.indexOf('noticeTitle'), 1);
+    }
+  }
+
+  /**
+   * States
+   */
+
+  /**
+   * State to track validation on properties
+   * Contains a list of properties that have an error associated with them
+   */
+  @State() errors: Array<string> = [];
+
+  /**
+   * Language of rendered component
+   */
+  @State() lang: string;
+
+  /*
+   * Observe lang attribute change
+   */
+  updateLang() {
+    const observer = new MutationObserver(mutations => {
+      if (mutations[0].oldValue != this.el.lang) {
+        this.lang = this.el.lang;
+      }
+    });
+    observer.observe(this.el, observerConfig);
+  }
+
+  private validateChildren() {
+    if (this.el.innerHTML.trim() == '') {
+      this.errors.push('children');
+    } else if (this.errors.includes('children')) {
+      this.errors.splice(this.errors.indexOf('children'), 1);
+    }
+  }
+
+  private validateRequiredProps() {
+    this.validateNoticeTitle();
+    this.validateType();
+    this.validateChildren();
+
+    if (
+      this.errors.includes('type') ||
+      this.errors.includes('noticeTitle') ||
+      this.errors.includes('children')
+    ) {
+      return false;
+    }
+    return true;
+  }
+
+  async componentWillLoad() {
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+
+    this.updateLang();
+
+    let valid = this.validateRequiredProps();
+
+    if (!valid) {
+      logError('gcds-notice', this.errors);
+    }
+  }
+
+  render() {
+    const { type, noticeTitle, lang } = this;
+
+    return (
+      <Host>
+        {this.validateRequiredProps() && (
+          <section class={`gcds-notice notice--type-${type}`}>
+            <gcds-heading tag="h2" marginTop="0" class="notice__heading">
+              <strong class="notice__type">
+                {i18n[lang]['type'][type]}
```

Comment:
What about if the web author want to use alternative words which do also provide a sense of the type of notice?

Take for example an impromptu office closure, IMHO the following notice wont be desirable as there is not necessary means an immediate danger for the public:

> Danger: Office closure
> The office X was closed for the day due to Y situation.
![image](https://github.com/user-attachments/assets/e0e435dd-b518-4607-9447-5e51d1919312)


May be they it will be preferable to have something more contextualized which should be rendered as a danger notice type like:

> Unplanned: Office closure
> The office X was closed for the day due to Y situation.
![image](https://github.com/user-attachments/assets/bea546ce-68ed-4e92-9b8d-b2c61c026ad3)

I would like to propose to add a new property to allow the web author to customization the text conveying the importance level of the notice. If not provided, it can fallback on this default.

Thought?

Reply:
Thank you for the input! We have made the decision to change the design of the notice component to back to the design already available from canada.ca (with the icon). We just haven't had a chance to update this PR yet with those design changes